docs/source/guides/network_node/staking_guide.rst

```diff
@@ -488,6 +490,17 @@ This can help to decrease gas consumption in some operations. To merge two stake
     (nucypher)$ nucypher stake merge --hw-wallet
 
 
+Remove unused sub-stake
+***********************
+
+Unused sub-stakes still cost some amount of gas during operations. 
```

Comment:
Technicaly this is wrong, because merging stakes will produce two sub-stakes and sometimes both of them will be active at least one period (when node is online at least 2 periods), also inactive sub-stakes will be shown if special flag will be specified. Anyway I'd prefer not to have here info about merge stake

Reply:
I had another go! But no pressure to make this change, I just think some context is required.

Reply:
✔️(changed one word)